specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -130,13 +130,17 @@
             "$ref": "#/parameters/billingAccountNameParameter"
           },
           {
-            "name": "parameters",
+            "name": "patch",
             "in": "body",
+            "description": "The payload that is used to patch the billing account.",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/BillingAccountUpdateRequest"
-            },
-            "description": "Request parameters supplied to the update billing account operation."
+              "uniqueItems": false,
```

Comment:
What is in master is compliant with ARM requirements for patch, the changes you're proposing aren't. 

https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#patch-resource

Can you not achieve your scenario using the resource schema, limited to only patchable properties?

Reply:
I had generated the DotNet SDK and the update thows bad request error since the Patch takes JsonPatchDocument.

Reply:
Even try now link on Docs site is broken, and when I follow the example in the document, it throws bad request. This is same for BillingAccount, BillingProfile and InvoiceSection PATCH operation. 
https://docs.microsoft.com/en-us/rest/api/billing/2019-10-01-preview/invoicesections/update